src/index_template.html

```diff
@@ -114,14 +231,46 @@ <h5 class="hero-subtitle">GeoPlatform is making Federal GeoData <span>F</span>in
             governments, private sector, academia, and the general public. <br>
           </p>
           <p>
-            <span><a href="#SUPPORT_URL#/gp-info/data-and-privacy-policies.html">Data and Privacy Policies</a></span>
+            <span><a href="https://sit-kb.geoplatform.info/gp-info/data-and-privacy-policies.html">Data and Privacy Policies</a></span>
```

Comment:
`<a href="#SUPPORT_URL#/gp-info/data-and-privacy-policies.html">Data and Privacy Policies</a>`